packages/toolpad-components/src/TextField.tsx

```diff
@@ -35,6 +35,7 @@ export default createComponent(TextField, {
       onChangeProp: 'onChange',
       defaultValue: '',
       defaultValueProp: 'defaultValue',
+      isScoped: true,
```

Comment:
I introduced the concept of scoped parameters as in Retool input values seem unique per scope but other props are not.